lib/helpers/updateCollection.js

```diff
@@ -2,11 +2,11 @@ import { EVENT_ACTIONS } from '../constants';
 
 const { DELETE, CREATE, UPDATE } = EVENT_ACTIONS;
 
-function sort(list = [], hasSort) {
-    if (!hasSort) {
+function sort(list = [], sortByKey) {
```

Comment:
should `sortByKey` have the default value `Order`?

Reply:
this function is only used in one place in this file, and `sortByKey` is `undefined` if objects in the list have neither `Order` nor `Priority` properties (in which case we don't sort the list at all)

Reply:
But right now having the property `Order` takes preference over having the property `Priority`. Which one is more important?